app/services/onboarding.js

```diff
@@ -0,0 +1,56 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import { TOAST_OPTIONS } from '../constants/toast-options';
+
+export default class OnboardingService extends Service {
+  @service store;
+  @service login;
+  @service toast;
+
+  constructor() {
+    super(...arguments);
+    this.login.checkAuth();
+  }
+  async signup(dataToUpdate) {
+    try {
+      let user = this.store.peekRecord('user', dataToUpdate.username);
```

Comment:
does this return null if no data is found?

Reply:
It's creating new user

Reply:
![image](https://github.com/Real-Dev-Squad/website-www/assets/111434418/2a823024-7ffa-4161-a412-a9f577c8ed46)
It returns only a record or I have applied a condition where there is no user then created a store it didn't return an empty object